hw06-atm/src/main/java/Bankomat.java

```diff
@@ -0,0 +1,9 @@
+import java.util.HashMap;
+
+public interface Bankomat {
```

Comment:
```suggestion
public interface ATM {
```

Не используйте транслит для имен.

https://translate.google.com/?sl=ru&tl=en&text=%D0%B1%D0%B0%D0%BD%D0%BA%D0%BE%D0%BC%D0%B0%D1%82&op=translate